src/FileIO.h

```diff
@@ -60,4 +61,26 @@ struct ScopedFileDescriptor {
       close(fd);
    }
 };
+
+// Do file access as root user for the function passed in.
+// Example: auto result = FileIO::SudoFile(FileIO::ReadAsciiFileContent, filePath);
```

Comment:
I'm missing something obvious. What is the reasoning with the "File" ending? Wouldn't "SudoAccess" or just "Sudo" be more appropriate?